### PR TITLE
Show affiliation of authors in abstract page

### PR DESCRIPTION
**Pull request by mvidalgarcia**
I don't know how to make the `background-color` to match the width of the user.
image

### CODE REVIEW

**Comment by ThiefMaster** [comment]
Do we even need a background color there?

**Comment by nop33** [comment]
I think it looks good with the bg. Doesn't `display: inline-block` fix the problem?

File: indico/htdocs/sass/modules/_users.scss [hunk 1]
```diff
@@ -78,10 +78,14 @@
     display: inline-block;
     font-size: 0.95em;
     padding: 0.2em 0.5em;
-    margin-bottom: 0.1em;
+    margin-bottom: 0.5em;
```
Comment (OmeGak) [inline indico/htdocs/sass/modules/_users.scss]: We don't want to increase `margin-bottom` in all cases. We are using `.event-user` in other places, like for instance in meeting pages ([example](https://indico.cern.ch/event/588588/)).

File: indico/htdocs/sass/modules/_users.scss [hunk 2]
```diff
@@ -78,10 +78,14 @@
...
     color: $black;
     background: $pastel-gray;
 
+    &.one-line {
```
Comment (OmeGak) [inline indico/htdocs/sass/modules/_users.scss]: I don't think this belongs in here. If you want to have one per line simply target `.author-list li` within the specific box in `_abstract_page.scss`.

**Comment by mvidalgarcia** [comment]
@nop33 I doesn't work with `display: inline-block` since we want each user in a different line as @pferreir suggested. With `inline-block` would look like this which IMO is not that bad.
image

**Pull request merged**